### PR TITLE
Internationalization support

### PR DESCRIPTION
**Pull request by fernandoalava**
This PR adds internationalization using https://www.i18next.com, right now the main idea is just to add internationalization. I tried don't touch too many function or interfaces

Next followup tasks:

- Add language flags button/select somewhere in the screen to change the default language (right now detect browser language)(design ideas are welcome)
- Add Spanish language support since it is my mother tongue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,13 @@
     "not op_mini all"
   ],
   "dependencies": {
+    "i18next": "^19.0.3",
+    "i18next-browser-languagedetector": "^4.0.1",
+    "i18next-xhr-backend": "^3.2.2",
     "nanoid": "2.1.9",
     "react": "16.12.0",
     "react-dom": "16.12.0",
+    "react-i18next": "^11.3.1",
```
Comment (lipis) [inline package.json]: pin versions

Comment (fernandoalava) [reply package.json]: 👍

Comment (lipis) [reply package.json]: and re run npm install

Comment (fernandoalava) [reply package.json]: Done, also delete my node_modules folder and then re-run npm install.

Comment (fernandoalava) [reply package.json]: oh I see, my PR does not refresh yet give me a minute

**Comment by dwelle** [comment]
Too bad the library uses a solution that isn't statically analyzable. So we'll need to do search & replace on every refactor.

**Comment by fernandoalava** [comment]
> Too bad the library uses a solution that isn't statically analyzable. So we'll need to do search & replace on every refactor.

Yes, that is the bad part, I choose this library because is a mature project perhaps there exist some others that could be a good idea to look at.

**Comment by vjeux** [comment]
Thanks! Let's merge it in and start doing your follow-ups (adding more languages and add a language selector)

**Comment by fernandoalava** [comment]
Perfect, I will ASAP 👌🏻

**Comment by vjeux** [comment]
One more follow up, could you check if there’s a way to tell the browser to prefetch the json files of the most common languages so we don’t wait the entire roundtrip before downloading the language files?

**Comment by fernandoalava** [comment]
Sure I will check 👍🏻

**Comment by lipis** [comment]
@fernandoalava Do you mind joining the project on Crowdin and updating the Spanish translations?

**Pull request merged**